superset-frontend/src/features/alerts/components/NotificationMethod.tsx

```diff
@@ -110,6 +123,18 @@ export const NotificationMethod: FunctionComponent<NotificationMethodProps> = ({
     }
   };
 
+  const onSubjectChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
+  ) => {
+    const { target } = event;
+
+    setSubjectValue(target.value);
```

Comment:
since you're passing the email subject down as a prop, you can use the prop value from the parent instead of storing it here.

Reply:
Modified